src/scripts/build_site.py

```diff
@@ -83,7 +83,7 @@ def read_info(filepath, to_dict=False):
 
 def get_option(comic_info: RawConfigParser, section: str, option: str, option_type: type=str, default: str=None) -> str:
     if comic_info.has_section(section):
-        if comic_info.has_option(section, option):
+        if comic_info.has_option(section, option) and comic_info.get(section, option):
```

Comment:
What is a use-case where you found this fix was needed? This change would make it impossible for a user to supply an empty string for a config option, if that config option had a hardcoded default. That said, I can't think of a situation where a user would want an empty string as their config value, but still...

Reply:
Sounds good! Removing it from the default file is a fine fix too if you think this may cause unexpected behavior.